### ISSUE TITLE
Settings

### ISSUE THREAD

**Issue by eileenmcnaughton** [totten/civix#29]
Should we be adding this into civix setup - declaration of settings folder (if exists)

``` php
/**
* (Delegated) Implementation of hook_civicrm_alterSettingsMetaData
*/
function _multisite_civix_civicrm_alterSettingsFolders(&$metaDataFolders = NULL) {
  static $configured = FALSE;
  if ($configured) return;
  $configured = TRUE;

  $extRoot = dirname( __FILE__ ) . DIRECTORY_SEPARATOR;
  $extDir = $extRoot . 'settings';
  if(!in_array($extDir, $metaDataFolders)){
    $metaDataFolders[] = $extDir;
  }
}
```

Link to eg. of extension setting file
https://github.com/eileenmcnaughton/org.civicrm.multisite/tree/master/settings

**Comment by totten** [comment]
Ultimately addressed by #150.

**Comment by eileenmcnaughton** [comment]
@totten perhaps after 5 years we should just keep this open for sentimental reasons :-)
